pkg/infra/plugin/plugin.go

```diff
@@ -58,7 +58,22 @@ func RenderCNI(cluster *scope.Cluster) ([]byte, error) {
 		return nil, errors.Wrapf(err, "failed to get cni yaml form %s", pluginFilename)
 	}
 
-	opts.PluginYAML = out
+	t := template.New("plugin template")
+	tpl, err := t.Funcs(sprig.TxtFuncMap()).Parse(out)
+	if err != nil {
+		return nil, err
+	}
+
+	var b bytes.Buffer
+	if err := tpl.Execute(&b, map[string]interface{}{
+		"ServiceCIDR": cluster.ServiceCIDR,
+		"PodCIDR":     cluster.PodCIDR,
+		"Extra":       cluster.CNIExtra,
```

Comment:
I can not find these three args in template, do you miss updating it

Reply:
refactor this, also add a UT for it.